### PR TITLE
[5.0] escape url in pagination bootstrap presenter

### PR DESCRIPTION
**Pull request by tormit**
Pager links are not escaped.
This corrupts links when some parameter matches with html entity like &reg.

### CODE REVIEW

File: src/Illuminate/Pagination/BootstrapThreePresenter.php [hunk 1]
```diff
@@ -76,7 +76,7 @@ protected function getAvailablePageWrapper($url, $page, $rel = null)
 	{
 		$rel = is_null($rel) ? '' : ' rel="'.$rel.'"';
 
-		return '<li><a href="'.$url.'"'.$rel.'>'.$page.'</a></li>';
+		return '<li><a href="'.htmlentities($url).'"'.$rel.'>'.$page.'</a></li>';
```
Comment (GrahamCampbell) [inline src/Illuminate/Pagination/BootstrapThreePresenter.php]: Why not use laravel's "e" function?

**Comment by taylorotwell** [comment]
Need more explanation.

**Comment by tormit** [comment]
The ampersands in urls must be escaped when used in links.  Otherwise your links may get corrupted.
Lets take an example.

Without escaping you have something like this in html:
```html
    <a href="http://example.com/path/?name=customer&reg_code=&search=1&submit=Search&page=2">2</a>
```

and this is what you get when you open the link(or hover it to see the actual url on status bar).
http://example.com/path/?name=customer®_code=&search=1&submit=Search&page=2
**Note that the _&reg_code_ parameter gets lost.**

Therefore the url must be escaped and problem above gets solved:
```html
   <a href="http://example.com/path/?name=customer&amp;reg_code=&amp;search=1&amp;submit=Search&amp;page=2">2</a>
```

**Comment by tormit** [comment]
I need to push my project into production soon, but this pagination bug needs to be fixed before. Please accept my fix.
I can make another commit if you want me to use Laravel e() function.

**Comment by AngryUbuntuNerd** [comment]
+1, just ran into the same issue

**Pull request merged**